go.mod

```diff
@@ -70,27 +71,28 @@ replace (
 	// Override version since auto-detected one fails with GOPROXY
 	github.com/census-instrumentation/opencensus-proto => github.com/census-instrumentation/opencensus-proto v0.2.0
 	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.3.1
+	github.com/prometheus/client_golang => github.com/prometheus/client_golang v0.9.4
```

Comment:
Is this automatically here or did you need to add it? Is it also because of GOPROXY?

Reply:
@michaelbeaumont I added it manually to avoid below issue during build.

```
../../../../pkg/mod/k8s.io/component-base@v0.16.8/metrics/legacyregistry/registry.go:44:9: undefined: prometheus.InstrumentHandler
```

k8s.io/kops (v1.15.2 or v1.16.2) is still using old version of component-base, so if we replace component-base, we need to replace its dependency (e.g. prometheus/client_golang) as well.

Latest alpha version of kops fixes this issue, but I reckon that it's better to wait for official release.

Reply:
oh man, golang dependency management :sweat_smile:

Reply:
haha yeah agreed that transitive dependencies for replace directive could be better :100: